x/auth/client/rest/sign.go

```diff
@@ -41,11 +41,8 @@ func SignTxRequestHandlerFn(cdc *codec.Codec, cliCtx context.CLIContext) http.Ha
 			return
 		}
 
-		txBldr := authtxb.TxBuilder{
-			ChainID:       m.ChainID,
-			AccountNumber: m.AccountNumber,
-			Sequence:      m.Sequence,
-		}
+		txBldr := authtxb.NewTxBuilder(utils.GetTxEncoder(cdc), m.AccountNumber,
+			m.Sequence, 200000, 1.0, false, m.ChainID, m.Tx.GetMemo(), m.Tx.Fee.Amount)
```

Comment:
Why use `200000` here? The `StdFee` should have the gas. We should use that (`m.Tx.Fee.Gas`).

Reply:
updated

Reply:
Opened #3176 to validate the req body